packages/react/plugins/component-testing/index.ts

```diff
@@ -66,6 +68,34 @@ export function nxComponentTestingPreset(
       specPattern: 'src/**/*.cy.{js,jsx,ts,tsx}',
       devServer: {
         ...({ framework: 'react', bundler: 'vite' } as const),
+        viteConfig: async () => {
+          const normalizedPath = lstatSync(pathToConfig).isDirectory()
```

Comment:
Do we need to access the fs, or can we check the extension like `.ts'` or `.js`?

Reply:
we can check for the if it's just a .ts/.js file instead of using `lstatSync()`, though unsure why we don't want to use fs?

```ts
const normalizedPath = ['.ts', '.js'].some((ext) => pathToConfig.endsWith(ext))
 ? pathToConfig
 : dirname(pathToConfig);
```